subsys/bluetooth/host/hci_raw.c

```diff
@@ -88,19 +130,162 @@ int bt_recv_prio(struct net_buf *buf)
 	return bt_recv(buf);
 }
 
+static int bt_h4_send(struct net_buf *buf)
+{
+	u8_t type;
+
+	type = net_buf_pull_u8(buf);
+
+	switch (type) {
+	case H4_CMD:
+		bt_buf_set_type(buf, BT_BUF_CMD);
+		break;
+	case H4_ACL:
+		bt_buf_set_type(buf, BT_BUF_ACL_OUT);
+		break;
+	default:
+		LOG_ERR("Unknown H4 type %u", type);
+		return -EINVAL;
+	}
+
+	LOG_DBG("buf %p type %u len %u", buf, bt_buf_get_type(buf), buf->len);
+
+	return 0;
+}
+
+static void bt_cmd_complete_ext(u16_t op, u8_t status)
+{
+	struct net_buf *buf;
+	struct bt_hci_evt_cc_status *cc;
+
+	if (status == BT_HCI_ERR_EXT_HANDLED) {
+		return;
+	}
+
+	buf = bt_hci_cmd_complete_create(op, sizeof(*cc));
+	cc = net_buf_add(buf, sizeof(*cc));
+	cc->status = status;
+
+	bt_recv(buf);
+}
+
+static u8_t bt_send_ext(struct net_buf *buf)
+{
+	struct bt_hci_cmd_hdr *hdr;
+	struct net_buf_simple_state state;
+	int i;
+	u16_t op;
+	u8_t status;
+
+	if (!cmd_ext) {
+		return status;
```

Comment:
huh? You're returning an uninitialised variable here.

Reply:
Fixed